src/ngx_http_lua_ssl_module.c

```diff
@@ -303,9 +322,15 @@ ngx_http_lua_ffi_update_last_ticket_decryption_key(SSL_CTX *ctx,
     pkey = &pkey[keys->nelts-1];
 
     dd("replace the last key");
-    ngx_memcpy(pkey->name, key, 16);
-    ngx_memcpy(pkey->aes_key, key + 16, 16);
-    ngx_memcpy(pkey->hmac_key, key + 32, 16);
+    if (ngx_strlen(key) == 48){
```

Comment:
Ditto. Need a space before `{`.

Also, can we avoid using null-terminated C strings in `key`? This `ngx_strlen()` function is an `O(n)` operation. We should pass in the string size explicitly via function arguments.